src/main/java/seedu/exercise/logic/parser/SuggestCommandParser.java

```diff
@@ -69,7 +66,8 @@ public SuggestCommand parse(String args) throws ParseException {
      * Parses arguments and returns SuggestPossibleCommand for execution
      */
     private static SuggestCommand parsePossible(ArgumentMultimap argMultimap) throws ParseException {
-        if (!argMultimap.arePrefixesPresent(PREFIX_OPERATION_TYPE) || !argMultimap.getPreamble().isEmpty()) {
+        assert(argMultimap.getPreamble().isEmpty());
```

Comment:
Why the sudden change to assertion here? This check is for user input error

Reply:
because preamble was already checked before hand in parse

Reply:
Okay.